dist/index.js

```diff
@@ -19309,7 +19309,7 @@ module.exports = {
  */
 function normalizeOutputKey(dataKey, upperCase = false) {
   let outputKey = dataKey
-    .replace(".", "__")
```

Comment:
IIRC we used to wait to update `dist/index.js` until release time?

Reply:
Yes you are correct. I can't remember why I started including this in PRs. But at one point I think I had a good reason... Maybe it was so the main ref was up-to-date in case people wanted to pull that in?

Reply:
Evidence of that here: https://github.com/hashicorp/vault-action/pull/522#issuecomment-2310331130

Reply:
Yeah I think the concern was that `main` could be unstable in between releases and could break people that were pointing to `@main`. But that was early days here, maybe we're not as worried about that now.